community-plugins.json

```diff
@@ -45,8 +45,18 @@
     "_comment": "NEW STRUCTURE FOR PLUGINS BELOW",
     "capabilities": [
       "memories",
-      "chat"
+      "chat",
+      "voice"
     ],
+    "voice": {
+      "speak": {
+        "model": "aura-helios-en"
+      },
+      "listen": {
+        "model": "nova-2-general",
+        "endpointing": 300
```

Comment:
Controls how long to wait before triggering response
- https://developers.deepgram.com/docs/endpointing